composer.json

```diff
@@ -257,7 +257,7 @@
     "repositories": {
         "ps_apiresources": {
             "type": "vcs",
-            "url": "https://github.com/PrestaShop/ps_apiresources.git"
+            "url": "https://github.com/jolelievre/ps_apiresources.git"
```

Comment:
Sure to change the related user ?

Reply:
Yep, it's temporary, as the module depends on the core end the core depends on the module (because it runs the module's tests) We have a circular dependency that forces us to temporarily use the module's fork branch to check that all tests are good

Then we merge the module's PR and finally we can update the core composer back to the original repository